src/util.ts

```diff
@@ -81,9 +81,9 @@ export function compareBy<T, U>(f: (v: T) => U): (a: T, b: T) => number {
 	return (a: T, b: T) => {
 		const fa = f(a);
 		const fb = f(b);
-		if (fa < fb) {
+		if (fa > fb) {
 			return -1;
-		} else if (fb < fa) {
+		} else if (fb > fa) {
```

Comment:
Ah, my bad - I misread the sort order in 30f120a4c91bf660fc5f2f24633830da235e47a9. But `compareBy` is also used for sorting in ascending order in `Settings.txt`, so this would break that. I think a better solution would be to add a `reverse` function and apply that in the cases where descending order is intended, see:
- https://github.com/wwWallet/wallet-frontend/pull/338

Reply:
No problem! Perhaps we could remove this commit: [f565e58](https://github.com/wwWallet/wallet-frontend/pull/335/commits/f565e58bec839eb879298156e000b72bef76d5e8) ?